src/main/kotlin/org/rust/ide/annotator/RsFormatMacroAnnotator.kt

```diff
@@ -188,7 +188,14 @@ private val formatParameterParser = Regex("""(?x) # enable comments
 private fun parseParameters(formatStr: RsLitExpr): ParseContext? {
     val literalKind = (formatStr.kind as? RsLiteralKind.String) ?: return null
     val rawTextRange = literalKind.offsets.value ?: return null
-    val (unescapedText, sourceMap, _) = parseRustStringCharacters(rawTextRange.substring(literalKind.node.text))
+    val (unescapedText, sourceMap) = if (formatStr.rawStringLiteral != null) {
```

Comment:
```suggestion
    val (unescapedText, sourceMap) = if (literalKind.node.elementType in RS_RAW_LITERALS) {
```

Reply:
Byte literals are not allowed as format strings though. I wonder if we have a test for that, I'll check.

Reply:
There was no check, so I added one. Instead of marking the byte string as an error, I just ignore the format macro, because we still get false positives in resolving the macros.